### PR TITLE
Fixed small typos throughout

### PR DESCRIPTION
**Pull request by hhaun**
Made the following line changes (mainly to experiment with editing in github!):
Line 62: tried to fill in incomplete sentence
Line 80: runs ---> run
Line 116: validating ----> validate
Line 124: added missing word ("defined FOR a special github repo)
Line 178: superfluous "we"
Line 192: added period

### CODE REVIEW

File: Process_guide.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ A good PR is one that tackles one individual problem in a fairly atomic way, and
 
 Developers should use their judgment in calling out (via the @user mechanism) to anyone on the project who might have a special interest in a specific PR, or whose input might be particularly valuable.
 
-Since all repositories must have testing enabled and integrated with the Github PR system (see below), a reviewer will indicate to the author any PR that breaks the test suite needs to be put in test-compliant state
+Since all repositories must have testing enabled and integrated with the Github PR system (see below), a reviewer will indicate to the author any PR that breaks the test suite and that needs to be put in a test-compliant state.
```
Comment (mlhenderson) [inline Process_guide.md]: This line edit seems fine.

File: Process_guide.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ The topic of software testing is large and complex, and in order to make meaning
 
 -   Unit tests for single repositories.
 
--   Continuous integration (CI) for all repos that runs unit test suites, hooked to Github for code review via Travis CI.
+-   Continuous integration (CI) for all repos that run unit test suites, hooked to Github for code review via Travis CI.
```
Comment (mlhenderson) [inline Process_guide.md]: This line edit also seems fine.

File: Process_guide.md [hunk 3]
```diff
@@ -207,17 +207,17 @@ It's very important, in order to keep an entire distributed team moving forward,
...
+On Github, we will create teams that correspond to each of the official teams in the KBase org chart. These teams will have ‘admin privileges’ (in Github parlance), which enable their members to create repos owned by that team. As a matter of policy, team members should check with their team leader before creating a new repository. They can always start work if needed on a personal repo and transfer ownership to the team afterwards if an experiment pans out.
 
 On Github, all team members can merge PRs, and should consider all their repos to be a common responsibility, engaging with code review on all of them.
 
-If additional areas of work in KBase require teams that cut across the existing Org chart divisions, the leads of areas 3-7 can either make them directly, or ask any of the owners team to make a new one.
+If additional areas of work in KBase require teams that cut across the existing Org chart divisions, the leads of areas 3-7 can either make them directly, or ask any of the owner's team to make a new one.
```
Comment (mlhenderson) [inline Process_guide.md]: This one should be reverted or modified before acceptance.  There is a github "owners" team, which this is referencing.  It is not a grammatical error.  We could say github "owners" team possibly.

File: Process_guide.md [hunk 4]
```diff
@@ -207,17 +207,17 @@ It's very important, in order to keep an entire distributed team moving forward,
...
 
 # Repository and license management on Github
 
-On Github, we will create teams that correspond to each of the official teams in the KBase org chart. These teams will have ‘admin privileges’ (in Github parlance), which enables their members to create repos owned by that team. As a matter of policy, team members should check with their team leader before creating a new repository. They can always start work if needed on a personal repo and transfer ownership to the team afterwards if an experiment pans out.
+On Github, we will create teams that correspond to each of the official teams in the KBase org chart. These teams will have ‘admin privileges’ (in Github parlance), which enable their members to create repos owned by that team. As a matter of policy, team members should check with their team leader before creating a new repository. They can always start work if needed on a personal repo and transfer ownership to the team afterwards if an experiment pans out.
```
Comment (mlhenderson) [inline Process_guide.md]: Seems fine.

File: Process_guide.md [hunk 5]
```diff
@@ -207,17 +207,17 @@ It's very important, in order to keep an entire distributed team moving forward,
...
 
--   Be as atomic as possible. Don't open one ticket for three different, unrelated problems that will likely require three different people working on them (note that JIRA allows for sub-issues that can be assigned to different people, if necessary). Ideally, one ticket is one problem that one person can fix and close. Obviously, sometimes what appears to be one problem on the surface may turn out to be much more complicated
+-   Be as atomic as possible. Don't open one ticket for three different, unrelated problems that will likely require three different people working on them (note that JIRA allows for sub-issues that can be assigned to different people, if necessary). Ideally, one ticket is one problem that one person can fix and close. Obviously, sometimes what appears to be one problem on the surface may turn out to be much more complicated.
```
Comment (mlhenderson) [inline Process_guide.md]: Looks good.

File: Process_guide.md [hunk 6]
```diff
@@ -189,7 +189,7 @@ Therefore, use the "golden rule": treat others as you would like to be treated.
...
+<span id="h.of49szsvxiyt" class="anchor"></span>As a starting point, we will take as our solution to use JIRA almost exclusively, unless there’s a very good reason to stick to Github issues for a specific repo. For example, the repo that contains this very document will require lots of early discussion that may be much easier to do directly on github as a combination of issues and PRs than by going all the way over to JIRA. So this repo may be one that uses issues, at least for the early discussion period. But in general, we’ll keep issues disabled on most repos and track all issues on JIRA. If this proves to not work well, we will reconsider and look for better solutions.
```
Comment (mlhenderson) [inline Process_guide.md]: Looks good.

File: Process_guide.md [hunk 7]
```diff
@@ -121,7 +121,7 @@ Each repository should report its CI build status at the top of its README file,
 
 KBase is complex enough that per-repository tests will never be sufficient to uncover problems that may arise due to the *interaction* between components. For this reason, we will also need to run systems-level tests in an environment that is as similar as possible to the development one that is used for staging all new work.
 
-ci.kbase.us will provide an environment for conducting integration tests. This system will update regularly against defined branches for each repo and will run integration tests defined a special github repo. Developers should add to this integration suite in the same way they do for unit-tests.
+ci.kbase.us will provide an environment for conducting integration tests. This system will update regularly against defined branches for each repo and will run integration tests defined for a special github repo. Developers should add to this integration suite in the same way they do for unit-tests.
```
Comment (mlhenderson) [inline Process_guide.md]: Probably should be "in" instead of "for" here.

File: Process_guide.md [hunk 8]
```diff
@@ -113,7 +113,7 @@ These are some potentially useful resources on the topic [FIXME: anyone can sugg
 
 Each repository’s unit test suite must be hooked up to a CI system that runs the entire test suite on every pull request and reports this information directly on the PR page.
 
-On Github, Travis CI provides this service, and it should be enabled for all of our repositories to help validating the status of each pull request. This helps the code review process, as reviewers can know before even starting, what impact the proposed changes will have on the test suite.
+On Github, Travis CI provides this service, and it should be enabled for all of our repositories to help validate the status of each pull request. This helps the code review process, as reviewers can know before even starting, what impact the proposed changes will have on the test suite.
```
Comment (mlhenderson) [inline Process_guide.md]: Agreed.

File: Process_guide.md [hunk 9]
```diff
@@ -175,7 +175,7 @@ In the open source scientific space, a good reference for a well documented proj
 
 ## Purpose
 
-By tracking all project activities at a high level in JIRA, we ease communication and coordination across the entire project, we can more easily detect bottlenecks and critical overload on individuals, and allocate resources more effectively.
+By tracking all project activities at a high level in JIRA, we ease communication and coordination across the entire project, can more easily detect bottlenecks and critical overload on individuals, and allocate resources more effectively.
```
Comment (mlhenderson) [inline Process_guide.md]: Seems fine.

**Comment by mlhenderson** [comment]
Holly, I made comments for each line you changed.  There were a couple edits that you should modify before your pull request can be accepted, but most of them were fine and improved the document.

File: Process_guide.md (hunk 7 above)
Comment (hhaun) [inline Process_guide.md]: I changed this to "in'

File: Process_guide.md (hunk 3 above)
Comment (hhaun) [inline Process_guide.md]: Thanks for the info. I changed back to original.

**Pull request merged**